blueprints/ember-nrg-ui/index.js

```diff
@@ -99,5 +106,5 @@ module.exports = {
       });
   },
 
-  normalizeEntityName: function () {},
+  normalizeEntityName: function () { },
```

Comment:
The linter wants you to delete this added space:
```suggestion
  normalizeEntityName: function () {},
```